sample_data_for_SIMSSA_DB/Location/add_locations.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+import csv
+from datetime import date
+import re
+print('the original cwd at Locations folder', os.getcwd())
```

Comment:
If you change this to a Django management command you don’t have to do any of the special setup to get the Django environment working….

https://docs.djangoproject.com/en/4.2/howto/custom-management-commands/